app/serializers/project_serializer.rb

```diff
@@ -1,6 +1,23 @@
 class ProjectSerializer < ActiveModel::Serializer
   # attributes to be serialized
-  attributes :id, :title, :course_id
+  attributes :id, :title, :course_id, :user_ids, :permissions
   # model association
   has_many :stories
+
+  def permissions
```

Comment:
@mvuajua  as you can see here, I am rendering the permissions like that. Is that fine for your needs?
The permission can change for every request, so I decided to output them every time.

Reply:
sounds good to me!

Reply:
@anjakammer tho I think read and create permissions don't really work there - those would need to be somewhere else. since if I can't read the thing, I won't ever get the permissions from it, and if I didn't create anything yet I'm also not getting the permissions.